gpkit/model.py

```diff
@@ -139,7 +138,7 @@ def allsubs(self):
     def signomials_et_al(self):
         "Get signomials, unsubbed, allsubs in one pass."
         signomials = self.signomials
-        unsubbed = NomialData(nomials=signomials)
+        unsubbed = self.unsubbed
```

Comment:
Nix! This is an extra line of code (and property) for slightly greater efficiency.